client/src/types/roles.ts

```diff
@@ -0,0 +1,21 @@
+export interface PermissionDetail {
+  read: boolean;
+  write: boolean;
+}
+
+export interface Permissions {
+  Inventory: PermissionDetail;
+  Security: PermissionDetail;
+  Customer: PermissionDetail;
+  Entry: PermissionDetail;
+  Form: PermissionDetail;
+}
```

Comment:
Any reason why these attributes aren't camelcase?

Reply:
They were like this in the database, I'd say they are sorta like enumerated keys so we are using a different pattern, we could possibly change it in the future if the team would like that.